chainer/functions/activation/log_softmax.py

```diff
@@ -108,15 +107,31 @@ def log_softmax(x, use_cudnn=True):
         ``log_softmax`` method is more stable.
 
     Args:
-        x (~chainer.Variable): Input variable.
-        use_cudnn (bool): If ``True``, cuDNN is enabled and cuDNN ver. 3 or
-            later is used, then this function uses cuDNN as the core
-            implementation.
+        X (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
```

Comment:
`X` should be lower-case because the input argument shown in the top line uses a lower-case letter.